fares-data-build-tool/fdbt-site
# feat: return validity page

# Description

-   This PR adds a new page into the return ticket user journey flow which collects information about the validity period of the return ticket.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that the new page appears as expected and behaves as expected.

# Type of change


# Checklist:
